pageserver/src/http/openapi_spec.yml

```diff
@@ -197,54 +197,6 @@ paths:
               schema:
                 $ref: "#/components/schemas/Error"
 
-  /v1/tenant/{tenant_id}/timeline/{timeline_id}/wal_receiver:
```

Comment:
We would need to enumerate new fields in the `TimelineInfo` schema in this file.
Also we should maintain it up to date with https://github.com/neondatabase/cloud/blob/main/swagger/pageserver.yml but for this case it's not super important, I guess.

Reply:
Go code generates API clients, if the methods from the clients were used, those could be affected.
In the end, it's extra fields added to JSON, so missing this update should not disrupt its work on the protocol level.